pyasic/miners/backends/marathon.py

```diff
@@ -302,3 +307,33 @@ async def _get_wattage_limit(
                 return web_miner_config["mode"]["concorde"]["power-target"]
             except LookupError:
                 pass
+
+    async def _get_pools(self, web_pools: dict = None) -> List[PoolMetrics]:
+        if web_pools is None:
+            try:
+                web_pools = await self.web.pools()
```

Comment:
Need to test this more, since sadly its not documented anywhere, they use it in the web interfeace.  Have a miner on the way that I can test this with, as with just the CB it doesnt seem to work.

Reply:
```
[{"index":0,"url":"stratum+tcp://ca.stratum.braiins.com:3333","user":"BR7609.S19k","status":"Alive","priority":0,"getworks":4,"accepted":52,"rejected":0,"discarded":0,"stale":0,"diff":"135355","diff1":1824777,"diffa":1824777,"diffr":0,"diffs":0,"lsdiff":8192,"lstime":"2.189571873s","latency":83,"block_height":858555,"submit_count":52,"connect_time":"1m15.189620331s","reconnect_count":1,"extra_nonce_change_count":0},{"index":1,"url":"stratum+tcp://us-east.stratum.braiins.com:3333","user":"BR7609.S19k","status":"Alive","priority":1,"getworks":5,"accepted":0,"rejected":0,"discarded":0,"stale":0,"diff":"512","diff1":0,"diffa":0,"diffr":0,"diffs":0,"lsdiff":1024,"lstime":"479084h37m20.189642998s","latency":63,"block_height":858555,"submit_count":0,"connect_time":"1m11.189657956s","reconnect_count":1,"extra_nonce_change_count":0},{"index":2,"url":"stratum+tcp://stratum.braiins.com:3333","user":"BR7609.S19k","status":"Alive","priority":2,"getworks":4,"accepted":0,"rejected":0,"discarded":0,"stale":0,"diff":"1024","diff1":0,"diffa":0,"diffr":0,"diffs":0,"lsdiff":8192,"lstime":"479084h37m20.189672498s","latency":62,"block_height":858555,"submit_count":0,"connect_time":"1m11.189684706s","reconnect_count":1,"extra_nonce_change_count":0}]
```

Heres a result from the pools web command.